### PR TITLE
Metric agg: drop "aggregated" from all names plus a bit of feedback

### PR DESCRIPTION
**Pull request by vitalyf007**

### CODE REVIEW

File: Test/CoreSDK.Test/Shared/DataContracts/MetricTelemetryTest.cs [hunk 1]
```diff
@@ -205,7 +225,48 @@ public void SerializeReplacesNaNStandardDeviationOn0()
 
             Assert.Equal(0, original.StandardDeviation.Value);
         }
+
+        [TestMethod]
+        public void SerializeReplacesNaNSumOn0()
```
Comment (olegsych) [inline Test/CoreSDK.Test/Shared/DataContracts/MetricTelemetryTest.cs]: Why do the names of tests for the `Sanitize` method start with _Serialize_?

File: src/Core/Managed/Shared/TelemetryClient.cs [hunk 2]
```diff
@@ -205,10 +205,9 @@ public void TrackMetric(string name, double value, IDictionary<string, string> p
         }
 
         /// <summary>
-        /// Send a <see cref="MetricTelemetry"/> for aggregation in Metric Explorer.
+        /// Send a <see cref="MetricTelemetry"/> for representing aggregated metric data.
         /// Create a separate <see cref="MetricTelemetry"/> instance for each call to <see cref="TrackMetric(MetricTelemetry)"/>.
         /// </summary>
-        [Obsolete("This method is obsolete. Use TrackAggregatedMetric method to send pre-aggregated metric data or MetricManager class to create metrics.")]
         [EditorBrowsable(EditorBrowsableState.Never)]
```
Comment (olegsych) [inline src/Core/Managed/Shared/TelemetryClient.cs]: Remove `[EditorBrowsable(Never)]` from `TrackMetric(MetricTelemetry)`?

File: Test/CoreSDK.Test/Shared/DataContracts/MetricTelemetryTest.cs (hunk 1 above)
Comment (vitalyf007) [inline Test/CoreSDK.Test/Shared/DataContracts/MetricTelemetryTest.cs]: Makes sense. Will change (the naming is historical but you bring a good point).

File: src/Core/Managed/Shared/TelemetryClient.cs (hunk 2 above)
Comment (vitalyf007) [inline src/Core/Managed/Shared/TelemetryClient.cs]: My bad :( Fixing.

Comment (olegsych) [inline src/Core/Managed/Shared/TelemetryClient.cs]: @vitalyf007 Thanks for removing `[EditorBrowseable]` from the `MetricTelemetry` class. I didn't catch that. Should we also remove it from the `TrackMetric(MetricTelemetry)` method now that it's no longer `[Obsolete]`?

Comment (vitalyf007) [inline src/Core/Managed/Shared/TelemetryClient.cs]: Yes.  Interesting that my Git client missed that change and only picked it up once I updated to the latest version...

**Pull request merged**